extern/covfie/CMakeLists.txt

```diff
@@ -13,7 +13,7 @@ message( STATUS "Fetching covfie as part of the traccc project" )
 
 # Declare where to get covfie from.
 set( TRACCC_COVFIE_SOURCE
-   "URL;https://github.com/acts-project/covfie/archive/refs/tags/v0.11.0.tar.gz;URL_MD5;9f41c9308e7801ac6319c470a1ec4212"
+   "URL;https://github.com/acts-project/covfie/archive/refs/tags/v0.12.1.tar.gz;URL_MD5;1512b389d8d7d08ef3bd30d6b18dcee1"
```

Comment:
I thought there was a separate Covfie update recently. Did that not go in yet? It would just seem better to factorize such things from each other.